### DISCUSSION TITLE
Question about the tree component

### DISCUSSION THREAD

**Discussion by ZVektor** [ant-design-blazor/ant-design-blazor#4300] [Q&A]
How to make an icon (picture, symbol) show on some nodes in the tree. For example, like this:
![image](https://github.com/user-attachments/assets/6825ffb9-1baa-4a39-a94a-c3c017956f9c)

Considering that I have a component without <Nodes><TreeNode> tags. Something like this:
```html
<Tree @ref="tree"
      DefaultExpandAll
      BlockNode
      ShowIcon DataSource="games"
      TitleExpression="x => x.DataItem.Text"
      ChildrenExpression="x => x.DataItem.Items"
      KeyExpression="x => x.DataItem.Id"
      TItem="GameElement">
</Tree>
```

**Comment by ElderJames** [comment]
You can use custom icon: https://antblazor.com/en-US/components/tree#components-tree-demo-customized-icon
